diff --git a/src/authority/message_request.rs b/src/authority/message_request.rs
index b2abb65..307458a 100644
--- a/src/authority/message_request.rs
+++ b/src/authority/message_request.rs
@@ -18,7 +18,7 @@ use crate::proto::{
 };
 
 /// A Message which captures the data from an inbound request
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Clone, Default)]
 pub struct MessageRequest {
     header: Header,
     query: WireQuery,
@@ -299,12 +299,21 @@ impl Queries {
 }
 
 /// A query with the original bytes stored from the query
-#[derive(Debug, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
 pub(crate) struct WireQuery {
     query: LowerQuery,
     original: Box<[u8]>,
 }
 
+impl Default for WireQuery {
+    fn default() -> Self {
+        Self {
+            query: LowerQuery::query(Default::default()),
+            original: Default::default(),
+        }
+    }
+}
+
 impl WireQuery {
     pub(crate) fn as_emit_and_count(&self) -> QueriesEmitAndCount<'_> {
         QueriesEmitAndCount {
